package.json

```diff
@@ -88,5 +88,34 @@
   "dependencies": {
     "@types/lodash.clonedeep": "^4.5.6",
     "lodash.clonedeep": "^4.5.0"
+  },
+  "exports": {
+    ".": "./index.js",
+    "./package.json": "./package.json",
+    "./actions/adjust": "./actions/adjust.js",
+    "./actions/customFunction": "./actions/customFunction.js",
```

Comment:
We're missing the entry point for the UMD bundle

`
Warning: Introducing the "exports" field prevents consumers of a package from using any entry points that are not defined, including the package.json (e.g. require('your-package/package.json'). This will likely be a breaking change.
`

looks like we need to specify *all* entrypoints.